api/v1/storagecluster_types.go

```diff
@@ -177,6 +177,9 @@ type ExternalStorageClusterStatus struct {
 	// GrantedCapacity Will report the actual capacity
 	// granted to the consumer cluster by the provider cluster.
 	GrantedCapacity resource.Quantity `json:"grantedCapacity,omitempty"`
+
+	// Id will describe the Id of an connected ExternalStorageCluster
+	Id string `json:"id,omitempty"`
```

Comment:
`Id` sounds a bit vague. It does not convey which ID is being referred.

Reply:
Suggestions:   `ProviderID` ,`OCSProviderID`,  `StorageClusterID`

Reply:
What do you think it should be? Can you list down the names you have in your mind?

Reply:
It should be `ConsumerId` as its the identifier for the consumer

Reply:
yeah. Ignore my above comment.  I thought its the storageCluster ID.  ConsumerID looks good.

Reply:
done